### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by zekkxx**
Made adjustments to the project package files including:
* Updates to dependencies to eliminate vulnerabilities
* Updates to contributor information and deployment information
* Reduction of unused NPM modules in packages

Made adjustments to the ReadMe documentation for more descriptive wording.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Final Project: Prota
-App for Project Managment
+# Prota
+A Full-Stack App for Agile development based Project Managment
 **Created**: `2019 June 09`
```
Comment (adj-dev) [inline README.md:4]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 ## About Prota
 A project management tool for GitHub developers - invite collaborators, develop projects, view sprints, and take on tasks in an agile-inspired service. 
 
 Deployed at: https://prota.herokuapp.com/
```
Comment (adj-dev) [inline README.md:8]:
```suggestion
**Created**: `2019 June 09`
Deployed at: https://prota.herokuapp.com/  
```

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
-# Final Project: Prota
-App for Project Managment
+# Prota
+A Full-Stack App for Agile development based Project Managment
 **Created**: `2019 June 09`
 
 ## About Prota
```
Comment (adj-dev) [inline README.md:5]:
```suggestion
## Prota
```

**Comment by zekkxx** [comment]
I like the changes Andrew! Excited to work on this again as well.
If anyone else has any ideas, please let me know. I hope you're all doing well!

**Pull request merged**